src/gateway/environments/jsdom-sixteen/jsdom-sixteen-environment.js

```diff
@@ -138,7 +138,7 @@ export class JSDOMSixteenEnvironment implements IRenderEnvironment {
                     } catch (e) {
                         const simplifiedError = extractError(e);
                         logger.error(
-                            `Closeable encountered an error during resource loader close: ${
+                            `Closeable encountered an error: ${
```

Comment:
This error message was being too specific since we're closing far more than the resource loader